ext/openAPI/hivemq-edge-openapi-master.yaml

```diff
@@ -4162,17 +4162,26 @@ paths:
       tags:
         - Topic Filters
   /api/v1/management/topic-filters/{filter}:
```

Comment:
is this actually the correct file? We use the one in the folder below 2025.4-SNAPSHOT for code generation